presto-main/src/main/java/com/facebook/presto/metadata/TablePropertyManager.java

```diff
@@ -69,9 +70,10 @@ public void addTableProperties(String catalog, List<PropertyMetadata<?>> tablePr
 
         // Fill in user-specified properties
         for (Map.Entry<String, Expression> sqlProperty : sqlPropertyValues.entrySet()) {
-            PropertyMetadata<?> tableProperty = supportedTableProperties.get(sqlProperty.getKey());
+            String property = sqlProperty.getKey().toLowerCase(ENGLISH);
```

Comment:
If we are doing this, we should verify in the `PropertyMetadata` constructor that the property is in lowercase.